### PR TITLE
tests: set a mtu lower than 1280

### PR DESCRIPTION
**Pull request by ffmancera**
This bug have been already fixed. This test checks that nmstate can set
a mtu lower than 1280 when IPv6 is not enabled.

Ref: https://bugzilla.redhat.com/1753128

Signed-off-by: Fernando Fernandez Mancera <ffmancera@riseup.net>

### CODE REVIEW

File: tests/integration/ethernet_mtu_test.py [hunk 1]
```diff
@@ -135,6 +135,12 @@ def test_decrease_to_lower_than_min_ipv6_iface_mtu(eth1_with_ipv6):
     assertlib.assert_state(original_state)
 
 
+def test_decrease_mtu_to_lower_than_1280(eth1_up):
```
Comment (tyll) [inline tests/integration/ethernet_mtu_test.py]: Since IPv6 being disabled is important here it should be mentioned in the test name, maybe `test_mtu_without_ipv6` and then parametrize it with 1279, 926 and 576 to test the edge cases and a middle value.

Even better would be to introduce a constant for 1280 such as `MINIMAL_IPV6_MTU` and then use `MINIMAL_IPV6_MTU - 1`.

Comment (ffmancera) [reply tests/integration/ethernet_mtu_test.py]: Done! Thanks!

**Comment by ffmancera** [comment]
Rebased!

File: tests/integration/ethernet_mtu_test.py [hunk 2]
```diff
@@ -135,6 +138,13 @@ def test_decrease_to_lower_than_min_ipv6_iface_mtu(eth1_with_ipv6):
     assertlib.assert_state(original_state)
 
 
+@pytest.mark.parametrize("mtu", [MINIMAL_IPV6_MTU - 1, 926, 576])
```
Comment (EdDev) [inline tests/integration/ethernet_mtu_test.py]: What are these `926, 576` mtu values represent?
These are not unit tests, why do we need to test these?

Comment (ffmancera) [reply tests/integration/ethernet_mtu_test.py]: After discuss this on IRC. We are only testing 576. Thanks!
